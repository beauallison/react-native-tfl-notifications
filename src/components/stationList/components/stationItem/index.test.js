import React from 'react';
import { shallow } from 'enzyme';
import StationItem from './index';

describe('components/stationList/components/stationItem', () => {
  it('should render a stationItem', async () => {
    const wrapper = shallow(<StationItem station="bakerloo" status="Good Service" />);

    expect(wrapper).toMatchSnapshot();
  });
});
