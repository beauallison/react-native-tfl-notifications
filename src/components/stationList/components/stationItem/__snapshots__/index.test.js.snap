// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/stationList/components/stationItem should render a stationItem 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <StationItem
    station="bakerloo"
    status="Good Service"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "chevronColor": "#bdc6cf",
      "disabled": false,
      "hideChevron": false,
      "leftIconUnderlayColor": "white",
      "rightIcon": Object {
        "name": "chevron-right",
      },
      "rightTitleNumberOfLines": 1,
      "roundAvatar": false,
      "selectedBackgroundColor": "grey",
      "subtitleNumberOfLines": 1,
      "switchButton": false,
      "textInputEditable": true,
      "title": "Bakerloo - Good Service",
      "titleNumberOfLines": 1,
      "underlayColor": "white",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "chevronColor": "#bdc6cf",
        "disabled": false,
        "hideChevron": false,
        "leftIconUnderlayColor": "white",
        "rightIcon": Object {
          "name": "chevron-right",
        },
        "rightTitleNumberOfLines": 1,
        "roundAvatar": false,
        "selectedBackgroundColor": "grey",
        "subtitleNumberOfLines": 1,
        "switchButton": false,
        "textInputEditable": true,
        "title": "Bakerloo - Good Service",
        "titleNumberOfLines": 1,
        "underlayColor": "white",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
